### ISSUE TITLE
Cannot create new HD account is last HD account is archived

### ISSUE THREAD

**Issue by tromer** [mycelium-com/wallet#334]
If all HD account are used but archived, trying to create a new HD account results in this error:
`You can only have one unused HD Account.`
(string `use_acc_first`).

If the last HD account is reactivated, then a new account can be created.

Version: Mycelium Wallet v2.8.6.

**Comment by Giszmo** [comment]
Yep. Way to reproduce is to create at least one active non-HD account. Sounds familiar ...

**Comment by jonathancross** [comment]
Please update title:
>Cannot create new HD account **_if_** last HD account is archived
